src/Provider/Provider.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -63,16 +59,53 @@ public async Task<DashboardViewModel> GetDashboardViewModelAsync(long ukprn, str
             {
                 Vacancies = vacanciesVm,
                 Pager = pager,
-                IsFiltered = filterStatus.HasValue,
-                ShowFilter = vacancies.Select(v => v.Status).Distinct().Count() > 1,
-                FilterOptions = GetFilterSelectOptions(vacancies, filterStatus),
+                IsFiltered = filterStatus.HasValue,                 
```

Comment:
We shouldn't need to set this.  We can have a property on VM `IsFiltered => Filter == FilterOptions.All`